examples/ansible/setup_grafana.yml

```
@@ -0,0 +1,49 @@
+---
+# Ansible Playbook: Setup Grafana on SUSE ALP Dolomite
+# Description: This Ansible playbook automates the deployment of Grafana on a SUSE ALP Dolomite host.
+# The steps include fetching the Grafana image, setting up the Grafana container, and providing access information.
+# The steps are based on https://documentation.suse.com/alp/dolomite/html/alp-dolomite/available-alp-workloads.html#task-run-grafana-with-podman
+
+- name: Setup ALP system for Grafana
+  hosts: alphost
+  become: true
+  vars:
+    workload:
+      name: grafana
+      image: registry.opensuse.org/suse/alp/workloads/tumbleweed_containerfiles/suse/alp/workloads/grafana:latest
+
+  tasks:
+    - name: Ensure Grafana image is present
+      containers.podman.podman_image:
+        name: "{{ workload.image }}"
+        state: present
+
+    - name: Retrieve image for workload {{ workload.name }}
+      ansible.builtin.command: >-
+        podman container runlabel install {{ workload.image }}
+      register: workload_runlabel_install
+      changed_when:
+        - ('already exist' not in workload_runlabel_install.stdout)
+
+    - name: Create the Grafana container from the downloaded image
+      ansible.builtin.command: >-
+        /usr/local/bin/grafana-container-manage.sh create
+      register: grafana_container_creation
+      changed_when:
+        - ('using /etc/grafana-container.conf as configuration file' in grafana_container_creation.stdout)
+
+    - name: Start the Grafana container
+      ansible.builtin.command: >-
+        /usr/local/bin/grafana-container-manage.sh start
+      register: grafana_container_start
+      notify: Display Grafana access information
+      changed_when:
+        - ('Up Less than a second' in grafana_container_start.stdout)
+
+  handlers:
```

```suggestion
      changed_when:
        - ('Up Less than a second' in grafana_container_start.stdout)

```

No need for handler, the debug message can be just displayed at the end of every successful run.